### PR TITLE
NXBT-3664: Update infrastructure changes

### PR DESCRIPTION
**Pull request by kevinleturc**

### CODE REVIEW

File: content/changelog.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ slug: "changes"
...
+- 2022-11-21 Use MongoDB 6.0.2 and Kafka 3.3.1 for LTS 2023
+
+
+- 2022-11-18 Use OpenSearch 1.3.6 instead of Elasticserach 7.9.2 for LTS 2023
+
+
+- 2022-10-03 Change benchmark infrastructure from AWS EC2 to GKE
+
+
+- 2021-03-22 Use Zulu JDK 11 instead of OpenJDK 11
```
Comment (ataillefer) [inline content/changelog.md]: Duplicate with the line below?

File: content/infrastructure.md [hunk 2]
```diff
@@ -4,6 +4,39 @@ slug: "infra"
 layout: "post"
 ---
 
+# Benchmarks æfter 2021
```
Comment (ataillefer) [inline content/infrastructure.md]: after

File: content/changelog.md (hunk 1 above)
Comment (bdelbosc) [inline content/changelog.md]: duplicate

File: content/infrastructure.md [hunk 3]
```diff
@@ -4,6 +4,39 @@ slug: "infra"
...
+- a Kafka cluster
+- a shared binary storage (S3 bucket)
+- a machine to run the bench (Jenkins pod)
+
+Everything is deployed with the help of Helmfile and Helm Charts, the definition and settings are accessible [here](https://github.com/nuxeo/nuxeo/blob/2021/ci/helm/helmfile.yaml) for the _benchmark_ environment.
+
+MongoDB and Elasticsearch are running on a dedicated Kubernetes node.
+
+There's one Kafka node for one Nuxeo node, both running on the same dedicated Kubernetes node.
+The number of nodes in the Nuxeo Cluster can be configured, default is 2, one application node and one worker node
```
Comment (ataillefer) [inline content/infrastructure.md]: missing final dot

File: content/infrastructure.md [hunk 4]
```diff
@@ -4,6 +4,39 @@ slug: "infra"
...
 ---
 
+# Benchmarks æfter 2021
+
+Nuxeo benchmark infrastructure runs on Google Kubernetes Engine (GKE).
+
+The benchmark infrastructure is composed of:
+- a MongoDB backend
+- a Elasticserach node (or Opensearch for LTS 2023)
+- a Kafka cluster
```
Comment (bdelbosc) [inline content/infrastructure.md:14]: - a Kafka cluster (brokers are on the same physical node than Nuxeo node)

File: content/infrastructure.md [hunk 5]
```diff
@@ -4,6 +4,39 @@ slug: "infra"
...
+- a shared binary storage (S3 bucket)
+- a machine to run the bench (Jenkins pod)
```
Comment (bdelbosc) [inline content/infrastructure.md]: /machine/node/ ?

File: content/infrastructure.md [hunk 6]
```diff
@@ -4,6 +4,39 @@ slug: "infra"
...
+
```
Comment (bdelbosc) [inline content/infrastructure.md:18]: - Nuxeo API (front) and worker nodes (number configurable)

File: content/infrastructure.md [hunk 7]
```diff
@@ -4,6 +4,39 @@ slug: "infra"
...
+Everything is deployed with the help of Helmfile and Helm Charts, the definition and settings are accessible [here](https://github.com/nuxeo/nuxeo/blob/2021/ci/helm/helmfile.yaml) for the _benchmark_ environment.
+
+MongoDB and Elasticsearch are running on a dedicated Kubernetes node.
+
+There's one Kafka node for one Nuxeo node, both running on the same dedicated Kubernetes node.
+The number of nodes in the Nuxeo Cluster can be configured, default is 2, one application node and one worker node
+
+Each Kubernetes node is composed of 16 vCPU and 64 GB Memory.
```
Comment (bdelbosc) [inline content/infrastructure.md]: maybe add the GCP name for such node

File: content/infrastructure.md [hunk 8]
```diff
@@ -4,6 +4,39 @@ slug: "infra"
...
+
+The benchmark is driven by continuous integration jobs (Jenkins).
+
+Here is a list of tools used to perform the benchmark:
+
+- [Helmfile](https://helmfile.readthedocs.io/): to deploy services.
+- [Gatling](http://gatling.io/): to generate load.
+- [gatling-report](https://github.com/nuxeo/gatling-report): home made open source tool to expose Gatling results.
+- [Datadog](https://www.datadoghq.com/): to monitor.
```
Comment (bdelbosc) [inline content/infrastructure.md]: Note that the Datadog monitoring is private.

File: content/changelog.md (hunk 1 above)
Comment (kevinleturc) [reply content/changelog.md]: Thanks, I was thinking about adding a line to talk about JDK 17 on LTS 2023.

Comment (kevinleturc) [reply content/changelog.md]: Thanks, I was thinking about adding a line to talk about JDK 17 on LTS 2023.

File: content/infrastructure.md (hunk 5 above)
Comment (kevinleturc) [reply content/infrastructure.md]: Changed to _a Jenkins pod to run the bench_.

File: content/infrastructure.md [hunk 9]
```diff
@@ -4,6 +4,39 @@ slug: "infra"
...
 ---
 
+# Benchmarks after 2021
+
+Nuxeo benchmark infrastructure runs on Google Kubernetes Engine (GKE).
+
+The benchmark infrastructure is composed of:
+- a Nuxeo cluster composed of API & Worker nodes (number configurable, default 1 of each)
+- a MongoDB backend
+- a Elasticserach node (or Opensearch for LTS 2023)
```
Comment (troger) [inline content/infrastructure.md]:
```suggestion
- a Elasticsearch node (or OpenSearch for LTS 2023)
```

**Pull request merged**